### PR TITLE
Gtable cleanup PTFM-16827

### PR DESCRIPTION
**Pull request by thanhhahaha**
Trying to remove GTable from the dx-toolkit helpstrings (all mentions of GTables on this page https://wiki.dnanexus.com/Command-Line-Client/Index%20of%20dx%20Commands). 

First commit in this branch just changed the helpstrings. Second commit attempts to remove the entire section of gtable-specific options from `dx head`. Not sure if removing the arguments from argparse is enough in this case ..

@ptn24 please review

### CODE REVIEW

**Comment by thanhhahaha** [comment]
Actually modified so we don't actually remove the gtable options in dx head but just unadvertised it. I also unadvertised dx export.

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ class DXCLICompleter():
                    'update': ['stage ', 'workflow ', 'org ', 'member ', 'project '],
                    'org': ['projects ', 'members ']}
 
-    silent_commands = set(['import'])
+    silent_commands = set(['import', 'export'])
```
Comment (ptn24) [inline src/python/dxpy/scripts/dx.py]: What are `silent_commands`?

**Comment by ptn24** [comment]
LGTM.

File: src/python/dxpy/scripts/dx.py (hunk 1 above)
Comment (thanhhahaha) [inline src/python/dxpy/scripts/dx.py]: Brett added this set in https://github.com/dnanexus/dx-toolkit/commit/d8408eba59cdbd4b439f01fbbec2c8c933fbcab0 to unadvertise all import commands. Now it unadvertises all export commands as well.

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -3758,25 +3758,29 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
...
-                                    help='Print part of a file or gtable',
-                                    description='Print the first part of a file or a gtable.  By default, prints the first 10 lines or rows, respectively.  Additional query parameters can be provided in the case of gtables.  The output for gtables is formatted for human-readability; to print rows in a machine-readable format, see "dx export tsv".',
+                                    help='Print part of a file',
+                                    description='Print the first part of a file.  By default, prints the first 10 lines.',
                                     parents=[no_color_arg, env_args],
                                     prog='dx head')
-parser_head.add_argument('-n', '--lines', type=int, metavar='N', help='Print the first N lines or rows (default 10)',
+parser_head.add_argument('-n', '--lines', type=int, metavar='N', help='Print the first N lines (default 10)',
                          default=10)
 head_gtable_args = parser_head.add_argument_group(title='GTable-specific options')
```
Comment (ptn24) [inline src/python/dxpy/scripts/dx.py]: Do we want to remove the reference to "GTable" here as well?

Comment (psung) [inline src/python/dxpy/scripts/dx.py]: FWIW after all the options in this category were marked as help-suppressed, this section header is no longer displayed.

**Comment by psung** [comment]
This was merged as b7891d715aba767b4dc49c9a294e16132f99a868

**Pull request closed without merging**